lib/decanter/parser/boolean_parser.rb

```diff
@@ -5,9 +5,23 @@ class BooleanParser < ValueParser
       allow TrueClass, FalseClass
 
       parser do |val, options|
-        raise Decanter::ParseError.new 'Expects a single value' if val.is_a? Array
-        next if (val.nil? || val === '')
-        [1, '1'].include?(val) || !!/true/i.match(val.to_s)
+        normalized_val = normalize(val)
+        next if normalized_val.nil?
+
+        true_values = ['1', 'true']
+
+        option_val = options.fetch(:true_value, nil)
```

Comment:
In the README, you named the option as `truth_value`. We'll need to update the code or the README to be consistent (and accurate)

Reply:
yep, this is a mistake. Do you think `truthy_value` or `truthy_values` is a better solution?